File: CS/DXApplication5/MyInplaceEditor.cs
namespace DXApplication5
{
    #region #myinplaceeditor
    public class MyInplaceEditor : DevExpress.XtraScheduler.Native.SchedulerInplaceEditorEx
    {
        public MyInplaceEditor(DevExpress.XtraScheduler.SchedulerInplaceEditorEventArgs inplaceEditorArgs)
            : base(inplaceEditorArgs)
        {
        }
        protected override void TextBox_KeyDown(object sender, System.Windows.Forms.KeyEventArgs e)
        {
            if (e.Control && e.KeyCode == System.Windows.Forms.Keys.S)
            {
                OnCommitChanges();
                e.Handled = true;
            }
            else if (e.Control && e.KeyCode == System.Windows.Forms.Keys.Z)
            {
                OnRollbackChanges();
                e.Handled = true;
            }
            else
                //base.TextBox_KeyDown(sender, e);
                e.Handled = true;
        }
    }
    #endregion #myinplaceeditor
}

File: VB/DXApplication5/MyInplaceEditor.vb
Imports DevExpress.XtraScheduler

Namespace DXApplication5
    #Region "#myinplaceeditor"
    Public Class MyInplaceEditor
        Inherits DevExpress.XtraScheduler.Native.SchedulerInplaceEditorEx

        Public Sub New(ByVal inplaceEditorArgs As DevExpress.XtraScheduler.SchedulerInplaceEditorEventArgs, ByVal apt As Appointment)
            MyBase.New(inplaceEditorArgs, apt)
        End Sub
        Protected Overrides Sub TextBox_KeyDown(ByVal sender As Object, ByVal e As System.Windows.Forms.KeyEventArgs)
            If e.Control AndAlso e.KeyCode = System.Windows.Forms.Keys.S Then
                OnCommitChanges()
                e.Handled = True
            ElseIf e.Control AndAlso e.KeyCode = System.Windows.Forms.Keys.Z Then
                OnRollbackChanges()
                e.Handled = True
            Else
                'base.TextBox_KeyDown(sender, e);
                e.Handled = True
            End If
        End Sub
    End Class
    #End Region ' #myinplaceeditor
End Namespace
